lib/mix/tasks/still.compile.ex

```diff
@@ -8,6 +8,7 @@ defmodule Mix.Tasks.Still.Compile do
 
     Still.Compiler.CompilationStage.subscribe()
 
+    Application.put_env(:still, :compiling, true)
```

Comment:
On the fence about calling it `compiling`, since with `mix still.dev` we might also be compiling. It just feels like it might be awkward for others. Maybe `compilation_task`? Open to suggestions here.

Reply:
👍